examples/storybook/package.json

```diff
@@ -1,46 +1,44 @@
 {
   "name": "storybook",
-  "version": "0.0.1",
+  "version": "0.0.0",
   "scripts": {
-    "start": "start-storybook -p 9001 -c .storybook",
-    "build": "build-storybook -c .storybook -o dist/ --quiet"
+    "docs:json": "compodoc -p ./tsconfig.json -e json -d .",
```

Comment:
do we need this?

Reply:
was automatically generated. I removed it and it still looks like it's working with out it. 
Apparently it's a tool that generated [documentation for Angular Modules](https://storybook.js.org/docs/riot/api/argtypes#automatic-argtype-inference) and expose the properties available and they types. Do you think we need this?

Reply:
done here 040e698, let's see if the build breaks 😄